### DISCUSSION TITLE
Comportamento inesperado builder.ToTable() em (relacionamento muitos para um)

### DISCUSSION THREAD

**Discussion by BrunoBafilli** [balta-io/forum#197] [Tira Dúvidas] [answered]
Boa tarde comunidade como vocês estão?

Estou com uma questão sobre o builder.ToTable() referente ao modulo do EF.

Quando estou efetuando um relacionamento do tipo MUITOS PARA UM, no caso 

```
builder.HasMany(x => x.Alunos)
                .WithOne(x => x.Escola)
                .IsRequired()
                .OnDelete(DeleteBehavior.Cascade);
```
Estou tendo uma ocorrencia inesperada.
 
O que é considerado MANY, por algum motivo, no banco de dados, essa tabela é criada plularizada, ou seja, com (S) no final, por exemplo.

Ao inves de a tabela conter o nome **Escola**, está ficando com o nome **Escolas**, isso ocorre mesmo eu setando a propriedade 
`builder.ToTable("Escola");`

Eu poderia forçar uma mudança diretamente nos arquivos gerados pelo migration, mas conforme boas práticas isso não é interessante.

Alguem saberia o motivo disso? esse de fato é o comportamento esperado?

Obrigado.

@andrebaltieri

**Comment by BrunoBafilli** [comment, marked as answer]
Problema resolvido.

Solução: Por algum motivo não estava sendo atualizada as minhas **migrations** quando eu executava um **database update**, deletei a **pasta migrations**, efetuei um novo **add migrations** e todas as **builder.ToTable()**; Voltaram ao comportamento esperado.

**Reply by andrebaltieri** [reply]
Obrigado por compartilhar a solução!
